zerver/webhooks/freshstatus/doc.md

```diff
@@ -0,0 +1,79 @@
+Receive Freshstatus notifications in Zulip!
+
+1. {!create-stream.md!}
+
+1. {!create-bot-construct-url-indented.md!}
+
+1. Go to your Freshstatus dashboard. Click **Settings** and then
+   **Integrations**. Go to **Webhooks** and click **Manage** and
+   then click **New Webhook**.
+
+1. Under **Webhook Name** enter `Zulip` or anything else you wish.
+   Enter whatever you like under **Description**.
+
+1. Under **Condition** you can set which events and components you would
+   like to alert for. Under **Action** ensure that **Request Type** is
+   set to `POST`. Under **Callback URL** enter the **URL** you
+   generated earlier.
```

Comment:
Under **Condition**, select the events you would like to be notified for. Under **Action**, set **Request Type** to `POST`. Set **Callback URL** to the URL constructed above.